.github/workflows/workload_checks.yml

```diff
@@ -0,0 +1,209 @@
+# Workload Checks Suite
+#
+# Runs Vector Workload Checks.
+#
+# Runs on:
+#  - scheduled daily UTC midnight
+
+# This workflow runs the collection of our workload checks, using the latest Vector nightly image,
+# which depends on when the workflow is invoked.
+#
+# The goal is to establish a baseline of check results for a variety of cases
+# and visualize trends for important Vector use cases.
+
+name: Workload Checks Suite
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Also ... is there any benefit to running this on the weekend? For example, the Component Features workflow we run 5 days a week.

Reply:
I don't see why not, since it will produce more data points.

Reply:
There's not any benefit that I'm aware of.

Reply:
Yeah naively I'd think we could change it to run Tue-Sat at 4am UTC, like the component features. Premise being that we wouldn't likely be pushing changes over the weekend to trigger nightly builds and thus the sat/sun runs of this workflow as-is wouldn't add anything over that Friday's run.